definitions/npm/redux-entitize_v0.2.x/flow_v0.45.x-/redux-entitize_v0.2.x.js

```diff
@@ -0,0 +1,48 @@
+import { type Schema } from "normalizr";
```

Comment:
This won't work as you'd expect it to...

Reply:
I realized, should have activated the linter to reveal the pointless import